src/map/skill.cpp

```diff
@@ -16221,9 +16218,7 @@ std::shared_ptr<s_skill_unit_group> skill_unitsetting(struct block_list *src, ui
 		 	val2 = sd->status.mother;
 		}
 		break;
-	case WE_CALLBABY:
-		if (sd) val1 = sd->status.child;
-		break;
+
```

Comment:
Revert this as it doesnt belong to any of the methods we're migrating (see skill_impl.hpp)